Aptoide/aptoide-client-v8
# [ASV-1297] Add catapult as default store

**NOTE:** **==>** **DO NOT ACCEPT IT UNTIL THE CATAPULT STORE IS CREATED** **<==**

**What does this PR do?**

This ticket should be assigned to Ana, after everything is reviewd in order to be later accepted when the catapult store is created.
The main goal of this PR is to add the catappult store as a default store. However there was three tasks in this ticket divided in the first three commits (not counting the merge one).
1st: Removed not used code related to defaultStoreName. (This variable was not being used in search and was left out due to the old partners build)
2nd: Main goal of the ticket
3rd: While doing the 2nd point, I noticed that we’re using get methods without them being necessary, since we could and should be using the injected fields. So in this commit aimed to removed the getMethods that had injected variables in AptoideApplication class 

**Database changed?**

  No

**Where should the reviewer start?**

AptoideApplication.java
SearchNavigator.java

**How should this be manually tested?**

  Run the app for the first and navigate to stores and check if the catappult store is there. (Since there’s no catapult store yet, go to the followed stores provider in applicationModule and change the catappult string for rmota).

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-1297

**Questions:**


**Code Review Checklist**
